docs/integration-testing-journey.md

```
@@ -302,11 +308,117 @@ _SPO2_PREFIX = "vital_signs/vital_signs:0/indirect_oximetry"
+      ├─ diastolic (DV_QUANTITY)
+      ├─ time (DV_DATE_TIME) - required
+      ├─ subject (PARTY_PROXY) - required
+      ├─ language (CODE_PHRASE) - required
+      └─ encoding (CODE_PHRASE) - required
+```
+
+**Note**: The `aqlPath` in the web template shows the full RM hierarchy including `/data[at0001]/events[at0006]/data[at0003]/items[...]`, but the FLAT format `id` fields suggest paths like `vital_signs/blood_pressure:0/systolic|magnitude`.
+
+### FLAT Format Testing Results
+
+**Test 1: Without Event Hierarchy**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)